gstreamer-plugin/git-patch/README.txt

```diff
@@ -8,11 +8,10 @@
 2. Apply SVT-HEVC plugin and enable libsvthevc to Gstreamer
 - git clone https://gitlab.freedesktop.org/gstreamer/gst-build.git
 - cd gst-build
-- git checkout 1.14
 - meson build
```

Comment:
is "git checkout 1.17" needed? or at lease tell user which version to use(master or which specific).

Reply:
I fixed that. thank you :)